extensions/query-store/src/extension.ts

```diff
@@ -10,7 +10,14 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 	// TODO: get db name
 	// TODO: add OE entry point with condition for command to only be visible for db's with Query Store enabled (or consider always showing and having a way to enable when dashboard is opened?)
 	// TODO: remove entry point from command palette - keeping for now to speed up testing so a connection doesn't need to be made to launch the dashboard
-	context.subscriptions.push(vscode.commands.registerCommand('queryStore.openQueryStoreDashboard', async () => { await new QueryStoreDashboard('AdventureWorks', context).open() }));
+	context.subscriptions.push(vscode.commands.registerCommand('queryStore.openQueryStoreDashboard', async (selectTab?: string) => {
```

Comment:
nit: "targetTab" maybe?  "selectTab" sounds more like verbNoun than adjectiveNoun to me

Reply:
updated to "targetTab". I initially had "selectTab" since that's what the api had, but they don't need to have the same name